### ISSUE TITLE
Apple Notes import not working as expected

### ISSUE THREAD

**Issue by traktuner** [standardnotes/forum#3571]
**Describe the bug**
I exported all my notes according to this guide:
https://standardnotes.com/help/31/how-can-i-import-my-notes-from-apple-notes
The "Export" app exports everything with the correct folder structure
When I then start the import process in Standard Notes and drag the exported folder into the import window it is stuck at
"Detecting service..."
Then nothing happens. Those are only about 400 notes.

When selecting "Plain text" in the import wizard I can only select individual notes, which is a bit of a hussle with my subfolder structure.

**Expected behavior**
Drag the exported Apple Notes folder into the Standard Notes import wizard and import all notes in all subfolders, preferably with the subfolder structure intact.

**Desktop (please complete the following information):**
 - OS: macOS 14.4.1 arm64
 - Version: 3.193.10

**Comment by vordenken** [comment]
Yeah same for me, wanted to switch to standard notes for cross platform e2e sync but importing all +300 notes manually is really not ideal.

**Comment by b6t** [comment]
Confirming the same behavior and the same post purchase displeasure...

Standard notes: Version 3.194.13 (3.194.13)
macos: 14.1 (23B74)

**Comment by b6t** [comment]
I emailed support on this. They said...

> I see, I'm very sorry to hear about this. However, I believe that if your resulting export from Apple notes included folders, I'm afraid you cannot drag the just the folder itself, because the app cannot import the actual folders into an account. It's important to note that the Folders on Standard Notes is actually an extension of the tags on the app, wherein you will be able to create sub tags by nesting them to create a folder-like hierarchy.
 
> Currently I'm afraid there is no workaround to immediately import your folder structure from Apple Notes, but in terms of importing the files, you should be able to click and drag the actual exported notes to the auto-detection box on the import tool, then the tool should be able to detect their format and proceed with the import to your account 🙏

**Comment by GeraldScott** [comment]
Further to the above, the same happened to me, except on Fedora 40 Desktop.

Steps to reproduce:

1. Get excited about an open source note app to replace Obsidian
2. Install using Flatpak
3. Register account
4. Find Import button (hidden bottom left)
5. Drag `*.md` from Obsidian folder to Import Drag-and-drop
6. Watch "Detecting Service.." message for 10 minutes
7. Give up and Google problem
8. Follow instructions [here](https://standardnotes.com/help/11/how-can-i-import-notes-from-a-bunch-of-text-files)
9. Use conversion tool to create `*.txt`
10. Drag `*.txt` file to Import Drag-and-drop
11. Watch "Detecting Service.." message for 5 minutes
12. Give up and try File picker
13. Watch "Detecting Service.." message for 5 seconds
14. Give up and try "Import from Plaintext/markdown"
15. Successfully import `*.txt`
16. Stare at JSON-formatted note in dismay
17. Delete and import original `*.md` using "Import from Plaintext/markdown"
18. All OK

There is definitely a defect in the Import Drag-and-drop window because the "Import from Plaintext/markdown" works without an issue.

But more broadly, please make (and document) the on-ramp from plain-text repos easier.

Thanks for all the good work.
